OpenSim/Actuators/Test/testMuscles.cpp

```diff
@@ -473,7 +450,7 @@ void simulateMuscle(
     values for K and dl/dt
 ============================================================================
 */    
-    if(testType == 1){
+    if(false){
```

Comment:
I'm not sure I follow; the initialization test was empty before and is also now empty?